targets/TARGET_STM/pwmout_api.c

```diff
@@ -39,6 +39,43 @@
 
 static TIM_HandleTypeDef TimHandle;
 
+/* Convert STM32 Cube HAL channel to LL channel */
+uint32_t TIM_ChannelConvert_HAL2LL(uint32_t channel, pwmout_t *obj)
+{
+#if !defined(PWMOUT_INVERTED_NOT_SUPPORTED)
+  if (obj->inverted) {
+    switch (channel) {
+    case TIM_CHANNEL_1  :
+      return LL_TIM_CHANNEL_CH1N;
+    case TIM_CHANNEL_2  :
+      return LL_TIM_CHANNEL_CH2N;
+    case TIM_CHANNEL_3  :
+      return LL_TIM_CHANNEL_CH3N;
+#if defined(LL_TIM_CHANNEL_CH4N)
+    case TIM_CHANNEL_4  :
+      return LL_TIM_CHANNEL_CH4N;
+#endif
+    default : /* Optional */
+      return 0;
+    }
+  } else
+#endif
+  {
+    switch (channel) {
+    case TIM_CHANNEL_1  :
+      return LL_TIM_CHANNEL_CH1;
+    case TIM_CHANNEL_2  :
+      return LL_TIM_CHANNEL_CH2;
+    case TIM_CHANNEL_3  :
+      return LL_TIM_CHANNEL_CH3;
+    case TIM_CHANNEL_4  :
+      return LL_TIM_CHANNEL_CH4;
+    default : /* Optional */
+      return 0;
```

Comment:
Your indentation is off here , 2 spaces compared to 4 in the rest of the file.

Reply:
Fixed